keysign/discover.py

```diff
@@ -42,7 +42,11 @@ def start(self):
         if self.bt_code and not self.stopped:
             # We try Bluetooth, if we have it
             log.info("Trying to connect to %s with Bluetooth", self.bt_code)
-            self.bt = BluetoothReceive(self.bt_port)
+            if self.bt is not None:
+                try:
+                    self.bt = BluetoothReceive(self.bt_port)
+                except ImportError:
```

Comment:
From your log I can see that the problem is that `BluetoothReceive` is None and a TypeError is raised because of this.
So there we should catch a TypeError instead of an ImportError, right?

Reply:
I thought about it, but I was inclined to the point that the user needs to install Pybluez and it's probably missing. I will correct it if that's what you want and add it to log.

Reply:
It's not that I want it. Here calling `BluetoothReceive` will never throw an `ImportError` because it has been already imported in the line 9.

Reply:
You are right. I messed up with a line.
I wrote my last comment answering your question. `ImportError`  because that's what you put when a python module was not installed. But anyways, you misunderstood because of the messed up line.